extensions/notebook/src/prompts/list.ts

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+// This code is originally from https://github.com/DonJayamanne/bowerVSCode
+// License: https://github.com/DonJayamanne/bowerVSCode/blob/master/LICENSE
+
+import { window } from 'vscode';
+import Prompt from './prompt';
+import EscapeException from './escapeException';
+
+export default class ListPrompt extends Prompt {
```

Comment:
Let's create a task to clean up the use of the Prompt framework given how little we use it. We can basically delete it & replace easily.

Reply:
https://github.com/Microsoft/azuredatastudio/issues/4100